tests/integration/actions/run/test_stdout.py

```diff
@@ -0,0 +1,30 @@
+""" run integration tests
```

Comment:
Thanks for adding the initial testcases for `run` subcommand. Since we can have tescases for EE and without EE the file can be renamed to `test_stdout_noee.py` in sync with testcases for other commands.

Reply:
Also will have to use the libtmux send keys to test for password prompts with local and container (EE) runs